src/components/sidebar/nodeView/NodeView.tsx

```diff
@@ -63,76 +90,114 @@ class NodeView extends React.Component
         // Empty
     }
 
-    public render(): any {
+    private capitalizeFirstLetter(input: string|null) {
+        if (!input) return '';
+        return input.charAt(0).toUpperCase() + input.slice(1).toLowerCase();
+    }
+
+    private renderNodeView(node: INode|null) {
         return (
-        <div className={s.nodeView}>
-            <ViewHeader
-                header='Karttakohde'
-            />
-            <div
-                className={classnames(
-                    s.flexRow,
-                    s.stopInUseRow,
-                )}
-            >
-                <div className={s.rowElement}>
-                    Pysäkki käytössä
-                </div>
-                <ToggleSwitch
-                    onClick={this.toggleStopInUse}
-                    value={true}
-                    type={TransitType.BUS}
-                    color={'#007ac9'}
-                />
-            </div>
-            <div className={s.flexRow}>
-                Sijainti: (2555744, 6675294)
-            </div>
-            <div className={s.flexInnerRow}>
-                <div className={s.flexInnerColumn}>
-                    <div>Karttatietolähde</div>
-                    <Dropdown
-                        onChange={this.onMapInformationSourceChange}
-                        items={this.state.mapInformationSource.items}
-                        selected={this.state.mapInformationSource.selected}
+            <div>
+                { node == null ? (
+                    <div>
+                        {this.props.nodeStore!.selectedNodeId ? (
+                            <div>Solmua {this.props.nodeStore!.selectedNodeId} ei löytynyt.</div>
+                        ) : (
+                            <div>Solmua ei löytynyt.</div>
+                        )}
+                    </div>
+                ) : (
+                <div>
+                    <ViewHeader
+                        header={`Solmu ${this.capitalizeFirstLetter(node.jore_solkuka)}`}
                     />
-                </div>
-            </div>
-            <div className={s.flexInnerRow}>
-                <div className={s.flexInnerColumn}>
-                    <div className={s.informationSource}>Tiedon lähde: pisteet</div>
+                    <div
+                        className={classnames(
+                            s.flexRow,
+                            s.stopInUseRow,
+                        )}
+                    >
+                        <div className={s.rowElement}>
+                            Pysäkki käytössä
+                        </div>
+                        <ToggleSwitch
+                            onClick={this.toggleStopInUse}
+                            value={true}
+                            type={TransitType.BUS}
```

Comment:
This type prop is actually not used in ToggleSwitch (atleast from what I found). So, we could probably remove it instead of setting a value that is just set without any reason.

Reply:
This was already done like this master but I can refactor it out.